pkgs/top-level/python-packages.nix

```diff
@@ -20519,6 +20585,20 @@ EOF
 
   magic-wormhole = callPackage ../development/python-modules/magic-wormhole { };
 
+  wsgiref = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.